package.json

```diff
@@ -84,6 +83,7 @@
     "steamladder": "^1.0.4",
     "transform-props": "^1.0.1",
     "turndown": "^6.0.0",
+    "unzip": "^0.1.11",
```

Comment:
what if we use https://www.npmjs.com/package/adm-zip?

Reply:
We could use that, it uses 0 dependencies so it's better than unzip